### PR TITLE
New release: v0.7.1-alpha

### PR DESCRIPTION
**Pull request by ellemouton**
Prepare the repo for a new minor release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +119,7 @@ The following table shows the supported combinations:
 
 | LiT              | LND          | Loop         | Faraday      | Pool         |
 |------------------|--------------|--------------|--------------|--------------|
+| **v0.7.1-alpha** | v0.15.0-beta | v0.19.1-beta | v0.2.8-alpha | v0.5.7-alpha |
```
Comment (guggero) [inline README.md:122]: I pushed a new tag for LNC: `v0.1.12-alpha`, maybe we can use that.

**Review by guggero — APPROVED** [review]
tACK, LGTM :tada:

**Comment by ellemouton** [comment]
cool, good idea @guggero - I added a new commit updating the go.mod to include the new LNC tag (i assume this is what you meant? or did you mean that I should add LNC client compatibility to the README matrix?). 
github auto-dismissed your review unforch 🙈

**Review by guggero — APPROVED** [review]
Yes, that's what I meant. There just was no diff in the go.mod that I could attach my comment to.

**Review by jamaljsr — APPROVED** [review]
tACK LGTM 👍

**Pull request merged**